### ISSUE TITLE
Changing of remarks colours for specific tags

### ISSUE THREAD

**Issue by eezj35** [nus-cs2103-AY2122S1/forum#365]
Hi prof @damithc , can i check, if my team were to reflect certain tags in a different colour from the rest, lets say if the tag contains the text "overdue", it will be red in colour, would this be considered a feature addition?

**Comment by damithc** [comment]
Yes @eezj35, it's a small feature, but a new feature nevertheless, assuming there is no color differentiation between tags in the current version.

**Comment by eezj35** [comment]
ok thanks prof!

> Yes @eezj35, it's a small feature, but a new feature nevertheless, assuming there is no color differentiation between tags in the current version.
ok thanks prof!

**Comment by charliemoweng** [comment]
Thank you prof @damithc for the clarification. Didn't expect the definition of "feature" to be this strict. 😅 I think our group need to examine our changes and be careful not to add a feature.

**Comment by crypto-code** [comment]
A follow up @damithc, for something similiar. Is it considered a feature if different tag types were already there and used in v1.3 and if the UI just gives different colors in the v1.4.

EDIT: The change has no functionality, it just enhances the UI.

**Comment by damithc** [comment]
> A follow up @damithc, for something similiar. Is it considered a feature if different tag types were already there and used in v1.3 and if the UI just gives different colors in the v1.4.
> 
> EDIT: The change has no functionality, it just enhances the UI.

Yes, it is against the feature freeze as it is not a fixing an existing bug and requires changes to the code that could potentially introduce more bugs.
